### PR TITLE
chore: add file with text for future features (Resolves #1053)

### LINKED ISSUE #588: Update copy
- [x] #750
- [x] #773
- [x] #775
- [x] #790
- [x] #780
- [x] #792
- [x] #853
- [x] #923
- [ ] #924
- [ ] #925
- [ ] #926
- [ ] #927
- [ ] #928
- [ ] #929
- [ ] #930
- [x] #931
- [x] #973
- [x] #1053

### LINKED ISSUE #926: Update text for Create and view AC/CC profile

### LINKED ISSUE #927: Update text for Create and view Comm Org profile

### LINKED ISSUE #928: Update text for Create and view FRO profile

### LINKED ISSUE #929: Update text for People and organizations

### LINKED ISSUE #930: Update text for Resources

### LINKED ISSUE #1053: Add placeholder page with text for features that haven't been implemented yet
In order to support translation work, it's necessary to have as many/all copy/text available early. As the strings are pulled from the source, we can temporarily add a template to house the missing copy so that they will be available for translations in the mean time.

### PR DESCRIPTION
**Pull request by jobara**
Adds a strings.php file which includes the translatable strings for future features (features to be implemented). This will allow the translation work to start before the features are implemented.

I've attempted to add in the tokens for cases where a value will be interpolated. I've also removed duplicates within the file, but have not checked existing text to see if there were any duplicates. Lastly I've added in markdown syntax where text formatting occurred with in a string, e.g. links, bold, etc.

Note: I've corrected a couple typos within the strings, but did not update the Figma wireframes.

Resolves #1053

Adds text string updates related to the following issues:

- #750 
- #926 
- #927 
- #928 
- #923 
- #929 
- #930 
- #931 

### Prerequisites

If this PR changes PHP code or dependencies:

- [x] I've run `composer format` and fixed any code formatting issues.
- [x] I've run `composer analyze` and addressed any static analysis issues.
- [x] I've run `php artisan test` and ensured that all tests pass.
- [x] I've run `composer localize` to update localization source files and committed any changes.

If this PR changes CSS or JavaScript code or dependencies:

- [x] I've run `npm run lint` and fixed any linting issues.
- [x] I've run `npm run build` and ensured that CSS and JavaScript assets can be compiled.

### CODE REVIEW

File: app/strings.php [hunk 1]
```diff
@@ -0,0 +1,1365 @@
...
+            __('Projects my organization is participating in as a Consultation Participant.'),
+            __('Projects my organization is contracted for as an Accessibility Consultant or Community Connector.'),
+            __('Projects I’m running'),
+            __('Projects my organization is running.'),
+            __('My trainings'),
+            __('All the training I am doing or have done.'),
+        ],
+        // https://www.figma.com/file/AXufBnFAVHvGdsxnQMVhNF/The-Accessibility-Exchange-Wireframes?node-id=2118%3A56436
+        'Dashboard/CommOrg/org admin filled' => [
+            __(':numRSVPs new rsvp’s to your engagement, :engagement'),
```
Comment (jobara) [inline app/strings.php]: Should `rsvp’s` be `RSVPs`?

File: app/strings.php [hunk 2]
```diff
@@ -0,0 +1,1365 @@
...
+        // https://www.figma.com/file/AXufBnFAVHvGdsxnQMVhNF/The-Accessibility-Exchange-Wireframes?node-id=1780%3A48437
+        'Dashboard/Ind-CP/Enter personal info-1' => [
+            __('Dashboard'),
+            __('Please enter your collaboration preferences'),
+            __('Matching information'),
+            __('Access needs'),
+            __('Language preferences'),
+            __('Payment information'),
+            __('Communication and consultation preferences'),
+            __('Optional'),
```
Comment (jobara) [inline app/strings.php:75]: I've assumed that `(Optional)` will be added separately, similar to the required flag.

File: app/strings.php [hunk 3]
```diff
@@ -0,0 +1,1365 @@
...
+    '0.0 Home Page' => [
+        // https://www.figma.com/file/AXufBnFAVHvGdsxnQMVhNF/The-Accessibility-Exchange-Wireframes?node-id=1125%3A3399
+        'HomePage' => [
+            __('Contrast'),
+            __('Back to top'),
+            __('About'),
+            __('Glossary'),
+            __('Privacy Policy'),
+            __('Available :start to :end Easter Standard Time'),
+            __('Available :start to :end :timeZone'),
```
Comment (jobara) [inline app/strings.php]: Added two examples as I'm not sure if the Time Zone will be interpolated.

File: app/strings.php [hunk 4]
```diff
@@ -0,0 +1,1365 @@
...
+    // Home Page - https://github.com/accessibility-exchange/platform/issues/750
+    '0.0 Home Page' => [
+        // https://www.figma.com/file/AXufBnFAVHvGdsxnQMVhNF/The-Accessibility-Exchange-Wireframes?node-id=1125%3A3399
+        'HomePage' => [
+            __('Contrast'),
+            __('Back to top'),
+            __('About'),
+            __('Glossary'),
+            __('Privacy Policy'),
+            __('Available :start to :end Easter Standard Time'),
```
Comment (greatislander) [inline app/strings.php]:
```suggestion
```

File: config/localizator.php [hunk 5]
```diff
@@ -26,7 +26,7 @@
         /**
          * Directories which should be looked inside.
          */
-        'dirs' => ['database/seeders', 'resources/views', 'app/Models', 'app/Http/Controllers', 'app/Http/Requests'],
+        'dirs' => ['database/seeders', 'resources/views', 'app/Models', 'app/Http/Controllers', 'app/Http/Requests', 'app'],
```
Comment (greatislander) [inline config/localizator.php]:
```suggestion
        'dirs' => ['database/seeders', 'resources/views', 'app'],
```

File: app/strings.php (hunk 3 above)
Comment (greatislander) [reply app/strings.php]: We'll want the interpolated version.

File: app/strings.php [hunk 6]
```diff
@@ -0,0 +1,1365 @@
...
+            __('Neurodivergence'),
+            __('Developmental'),
+            __('Learning'),
+            __('Cognitive'),
+            __('Mental health'),
+            __('Multiple disabilities'),
+            __('Episodic and invisible disabilities'),
+            __('DeafBlind'),
+            __('Body differences including size, limb and facial differences'),
+            __('Temporary disabilities'),
```
Comment (greatislander) [inline app/strings.php]: These should be removed as they exist in seeders.

File: app/strings.php [hunk 7]
```diff
@@ -0,0 +1,1365 @@
...
+            __('Other disability'),
+            __('Supporters'),
+            __('Virtual or in-person'),
```
Comment (greatislander) [inline app/strings.php:217]: These should be removed in favour of the options in `App\Enums\MeetingType`

File: app/strings.php [hunk 8]
```diff
@@ -0,0 +1,1365 @@
...
+            __('Project by :organization'),
+        ],
+        // https://www.figma.com/file/AXufBnFAVHvGdsxnQMVhNF/The-Accessibility-Exchange-Wireframes?node-id=1125%3A5877
+        'create new project - 3' => [
+            __('Steps for creating your project'),
+            __('Not started'),
+            __('Preview'),
+            __('Publish project'),
+            __('Once you publish your project, others can find it.'),
+            __('French translation of project goals'),
```
Comment (greatislander) [inline app/strings.php]: I think all of these types of strings need to be replaced with `:language translation of project goals`, as we can interpolate the language in a consistent manner

File: app/strings.php [hunk 9]
```diff
@@ -0,0 +1,1365 @@
...
+            __('Pain-related'),
+            __('(such as chronic fatigue syndrome)'),
+            __('(includes individuals with no spoken or signed language who communicate using gestures, pictures, letter boards, communication devices or assistance from a person who knows them well)'),
+            __('(such as Autism, ADHD)'),
+            __('(includes intellectual disability)'),
+            __('(such as dyslexia)'),
+            __('(includes traumatic brain injury, memory difficulties, dementia)'),
+            __('Mental health-related'),
+            __('(such as dual diagnosis of a mental health barrier, substance dependence)'),
+            __('(such as environmental, HIV, migraine)'),
```
Comment (greatislander) [inline app/strings.php]: These are already in `Database/Seeders/DisabilityTypeSeeder`

File: app/strings.php [hunk 10]
```diff
@@ -0,0 +1,1365 @@
...
+            __('Indigenous'),
+            __('Race and ethnicity'),
+            __('Immigrants and/or refugees'),
+            __('First language'),
+            __('Living in urban, rural, or remote areas'),
+            __('Notice'),
+            __('Please note that if you want to reach a specific identity group, you will not be able to use our matching service. You can reach them through an open call project, or go through a Community Connector or community organization.'),
+            __('Visual (including sight loss, blind, and partially sighted)'),
+            __('Women'),
+            __('Non-binary, gender non-conforming, gender fluid'),
```
Comment (greatislander) [inline app/strings.php]: These are already in `DisabilityTypeSeeder` and `GenderIdentitySeeder` (the last one is in `resources/views/organizations/edit/2.blade.php`)

File: app/strings.php [hunk 11]
```diff
@@ -0,0 +1,1365 @@
...
+            __('Prairie'),
+            __('Central'),
+            __('Atlantic'),
+            __('Northern'),
+            __('Groups'),
+            __(':percentage of your participants fall into more than one of these groups.'),
+            __('People with disabilities'),
+            __('Disabilities included'),
+            __(':percentage of your participants have more than one disability.'),
+            __('Visual'),
```
Comment (greatislander) [inline app/strings.php]: in `DisabilityTypeSeeder`

File: app/strings.php [hunk 12]
```diff
@@ -0,0 +1,1365 @@
...
+            __('2SLGBTQIA+'),
+            __('Trans'),
+            __('Black'),
+            __('Asian'),
+            __('Middle Eastern'),
+            __('White'),
+            __('Temporary residents, immigrants, and refugees'),
+            __('Single parents'),
+            __('Unemployed or underemployed'),
+            __('Financially precarious'),
```
Comment (greatislander) [inline app/strings.php]: Already present in seeders

File: app/strings.php [hunk 13]
```diff
@@ -0,0 +1,1365 @@
...
+            __('Your participants are from the following age groups.'),
+            __('Languages'),
+            __('Your participants use the following languages.'),
+            __('English'),
+            __('French'),
+            __('ASL'),
+            __('LSQ'),
```
Comment (greatislander) [inline app/strings.php]: We have localized language names already provided in a Composer package

File: app/strings.php [hunk 14]
```diff
@@ -0,0 +1,1365 @@
...
+            __('Telecommunication'),
+            __('Broadcasting'),
+            __('Federal government programs and services'),
+            __('Crown corporation'),
+            __('Employment'),
+            __('Buildings and public spaces'),
+            __('Information and communication technology'),
+            __('Communication, other than information and communication technologies'),
+            __('Buying goods and services'),
+            __('Providing programs and services'),
```
Comment (greatislander) [inline app/strings.php]: These are in `SectorSeeder` and `ImpactSeeder` already

File: app/strings.php [hunk 15]
```diff
@@ -0,0 +1,1365 @@
...
+        'Explore/CC' => [
+            __('Ethnoracial group they can connect to'),
+            __('First Nations'),
+            __('Inuit'),
+            __('Métis'),
+            __('East Asian'),
+            __('South Asian'),
+            __('Southeast Asian'),
+            __('Latin American'),
+            __('African'),
```
Comment (greatislander) [inline app/strings.php]: These are already present in `IndigenousIdentitySeeder` and `EthnoracialIdentitySeeder`

File: app/strings.php [hunk 16]
```diff
@@ -0,0 +1,1365 @@
...
+            __('Other'),
+            __('Other identity groups they can connect to'),
+            __('Non-binary, gender non-conforming, gender fluid people'),
+            __('Trans people'),
+            __('Immigrants and refugees'),
```
Comment (greatislander) [inline app/strings.php]: These are already in `ConstituencySeeder`

File: app/strings.php [hunk 17]
```diff
@@ -0,0 +1,1365 @@
...
+            __('Language groups they can connect to'),
+            __('Age groups they can connect to'),
+            __('Children (under 15)'),
+            __('Youth (15-30)'),
+            __('Working age adults (15-64)'),
+            __('Older people (65+)'),
```
Comment (greatislander) [inline app/strings.php]: These are in `AgeBracketSeeder`

File: app/strings.php [hunk 18]
```diff
@@ -0,0 +1,1365 @@
...
+            __('Here you may find resources helpful for participating in a consultation.'),
+            __('Browse resources for Consultation Participants'),
+            __('Community Connectors and Accessibility Consultants'),
+            __('Here you may find resources of benefit to Accessibility Consultants and Community Connectors.'),
+            __('Browse resources for Community Connectors and Accessibility Consultants'),
+            __('Here you may find resources to help support your members in participating in consultations.'),
+            __('Browse resources for community organizations'),
+            __('Here you may find resources to assist in running inclusive and accessible consultations.'),
+            __('Browse resources for Regulated Organizations'),
+            __('[CCRW Tools]'),
```
Comment (greatislander) [inline app/strings.php]: This should be removed, it's a placeholder

File: app/strings.php [hunk 19]
```diff
@@ -0,0 +1,1365 @@
...
+            __('Formats'),
+            __('Resource type'),
+            __('Phase of consultation'),
+            __('Sector'),
+            __('Area of impact'),
+            __('Format: :format'),
+        ],
+        // https://www.figma.com/file/AXufBnFAVHvGdsxnQMVhNF/The-Accessibility-Exchange-Wireframes?node-id=1125%3A13370
+        'Resources Hub/Search results' => [
+            __('12 results for “:search” found'),
```
Comment (greatislander) [inline app/strings.php]: Interpolate a result count here

File: app/strings.php [hunk 20]
```diff
@@ -0,0 +1,1365 @@
...
+            __('Which Indigenous groups can you connect to?'),
+            __('Which groups can you connect to?'),
+            __('People who identify with one or more of the 2SLGBTQIA+ identities'),
+            __('Which age groups can you connect to?'),
+            __('Older people (65 and over)'),
+            __('Which ethno-racial identity or identities are the people you can connect to?'),
+        ],
+        // https://www.figma.com/file/AXufBnFAVHvGdsxnQMVhNF/The-Accessibility-Exchange-Wireframes?node-id=1725%3A47554
+        'Consultant Community Connector / create profile - 8' => [
+            __('French translation of your lived experience'),
```
Comment (greatislander) [inline app/strings.php]: Interpolate languages here

**Review by greatislander — CHANGES_REQUESTED** [review]

File: app/strings.php (hunk 1 above)
Comment (greatislander) [reply app/strings.php]: yes please

File: app/strings.php [hunk 21]
```diff
@@ -0,0 +1,1365 @@
...
+            __('Community Connectors and Accessibility Consultants'),
+            __('Here you may find resources of benefit to Accessibility Consultants and Community Connectors.'),
+            __('Browse resources for Community Connectors and Accessibility Consultants'),
+            __('Here you may find resources to help support your members in participating in consultations.'),
+            __('Browse resources for community organizations'),
+            __('Here you may find resources to assist in running inclusive and accessible consultations.'),
+            __('Browse resources for Regulated Organizations'),
+            __('[CCRW Tools]'),
+            __('Here you may find resources for employees of the Regulated Organizations.'),
+            __('Browse resources by CCRW'),
```
Comment (jobara) [inline app/strings.php]: @greatislander is CCRW meaning full or should that be interpolated here?

**Comment by jobara** [comment]
@greatislander I've addressed your comments, and left one more question above.

File: app/strings.php (hunk 21 above)
Comment (greatislander) [reply app/strings.php]: It should be interpolated, thanks!

File: app/strings.php (hunk 1 above)
Comment (greatislander) [inline app/strings.php]:
```suggestion
            __(':numRSVPs new RSVP’s to your engagement, :engagement'),
```

File: app/strings.php [hunk 22]
```diff
@@ -0,0 +1,1244 @@
...
+            __('Browse resources for Consultation Participants'),
+            __('Community Connectors and Accessibility Consultants'),
+            __('Here you may find resources of benefit to Accessibility Consultants and Community Connectors.'),
+            __('Browse resources for Community Connectors and Accessibility Consultants'),
+            __('Here you may find resources to help support your members in participating in consultations.'),
+            __('Browse resources for community organizations'),
+            __('Here you may find resources to assist in running inclusive and accessible consultations.'),
+            __('Browse resources for Regulated Organizations'),
+            __('Here you may find resources for employees of the Regulated Organizations.'),
+            __('Browse resources by CCRW'),
```
Comment (greatislander) [inline app/strings.php]:
```suggestion
            __('Browse resources by :creator'),
```

**Review by greatislander — CHANGES_REQUESTED** [review]

**Review by greatislander — APPROVED** [review]

**Pull request merged**